pkg/cli/tag/tag.go

```diff
@@ -111,6 +113,7 @@ func NewCmdTag(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.C
 	cmd.Flags().BoolVar(&o.scheduleTag, "scheduled", o.scheduleTag, "Set a container image to be periodically imported from a remote repository. Defaults to false.")
 	cmd.Flags().BoolVar(&o.insecureTag, "insecure", o.insecureTag, "Set to true if importing the specified container image requires HTTP or has a self-signed certificate. Defaults to false.")
 	cmd.Flags().StringVar(&o.referencePolicy, "reference-policy", SourceReferencePolicy, "Allow to request pullthrough for external image when set to 'local'. Defaults to 'source'.")
+	cmd.Flags().StringVar(&o.importMode, "import-mode", o.importMode, "Imports the full manifest list of a tag when set to 'PreserveOriginal'. Defaults to 'Legacy'.")
```

Comment:
It is better to add an explanation like in here https://github.com/openshift/oc/pull/1289/files#diff-f0e0ab5ccc1ab1ecac81c724a4c086b8e9d16f525c0d23a91b37d8b2b10b365fR55

Reply:
Thanks for the feedback. I added an explanation. I wasn't sure whether `openshift/origin-control-plane:latest` is a good example here, because it doesn't have a manifest list. However, IMHO it's easier to parse the explanation as a user because it's the same as in the other examples (and they probably wouldn't tag this specific image anyways).